compiler/resolution/fixupExports.cpp

```diff
@@ -35,6 +36,8 @@
 static std::map<FnSymbol*, FnSymbol*> wrapperMap;
 static std::map<const char*, FnSymbol*> conversionCallMap;
 
+std::set<FnSymbol*> exportedStrRets;
```

Comment:
The wrapperMap above (which might be more aptly named as `wrapperToUnwrappedMap` is map with keys that are the exported wrapper functions, so you could achieve the same result by just iterating over the keys instead of introducing a set, if you want.